examples/postgresql/index.js

```diff
@@ -75,6 +75,12 @@ app.get('/secret', app.oauth.authorise(), function (req, res) {
   res.send('Secret area');
 });
 
+app.get('/scoped', app.oauth.authorise(), app.oauth.scope('demo'),
+    function (req, res) {
```

Comment:
This is just personal preference but I would prefer removing the newline before `function` here, to match the rest of the examples.